tests/integration/components/sl-calendar-test.js

```diff
@@ -0,0 +1,1461 @@
+        'JanFebMarAprMayJunJulAugSepOctNovDec',
+        'Twelve months are listed in order'
+    );
+});
+
+test( 'Twelve Years are Displayed in Order', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear', currentYear );
+
+    this.render( hbs`
+        {{sl-calendar
+            viewMode="years"
+            currentYear=currentYear
+        }}
+    ` );
+
+    let yearsSelection = this.$( '>:first-child' ).find( 'td' ).text().trim();
+    yearsSelection = yearsSelection.replace( /\s+/g, '' );
+
+    assert.equal(
+        yearsSelection,
+        '201920202021202220232024202520262027202820292030',
+        'Twelve years are listed in order'
+    );
+});
+
+// ******************************
+// Start of Dual Component Tests
+// ******************************
+
+test( 'Dual instance: Action fires when day is clicked', function( assert ) {
+
+    const currentYear1 = 2035;
+    this.set( 'currentYear1', currentYear1 );
+
+    const currentMonth1 = 1;
+    this.set( 'currentMonth1', currentMonth1 );
+
+    const currentYear2 = 2022;
+    this.set( 'currentYear2', currentYear2 );
+
+    const currentMonth2 = 9;
+    this.set( 'currentMonth2', currentMonth2 );
+
+    this.set( 'content', testContent );
+
+    this.render( hbs`
+        {{sl-calendar
+            action="testAction1"
+            content=content
+            currentYear=currentYear1
+            currentMonth=currentMonth1
+        }}
+        {{sl-calendar
+            action="testAction2"
+            content=content
+            currentYear=currentYear2
+            currentMonth=currentMonth2
+        }}
+    ` );
+
+    this.on( 'testAction1', () => {
+        assert.ok(
+            true,
+            'The test action 1 was called'
+        );
+    });
+
+    this.on( 'testAction2', () => {
+        assert.ok(
+            true,
+            'The test action 2 was called'
+        );
+    });
+
+    this.$( '>:nth-child(2) .active' ).click();
+});
+
+test( 'Dual instance: Navigating Forward by Month', function( assert ) {
+
+    const currentYear = 2022;
+    this.set( 'currentYear1', currentYear );
+    this.set( 'currentYear2', currentYear );
+
+    const currentMonth = 9;
+    this.set( 'currentMonth1', currentMonth );
+    this.set( 'currentMonth2', currentMonth );
+
+    this.render( hbs`
+        {{sl-calendar
+            currentMonth=currentMonth1
+            currentYear=currentYear1
+        }}
+        {{sl-calendar
+            currentMonth=currentMonth2
+            currentYear=currentYear2
+        }}
+    ` );
+
+    assert.equal(
+        this.$( '>:first-child .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance one: current month is set correctly'
+    );
+
+    assert.equal(
+        this.$( '>:nth-child(2) .datepicker-switch' ).text().trim(),
+        'September ' + currentYear,
+        'Component instance two: current month is set correctly'
+    );
+
```

Comment:
Are either of the previous two assertions needed?  Other tests in this suite have ensured this functionality, and since they year and month values are not being set to different ones it's not like we're checking for any bleed out.